app/src/main/java/com/pysiak/simple/clock/ui/ClockActivity.java

```diff
@@ -8,15 +8,21 @@
 import android.support.v7.app.AppCompatActivity;
 import android.widget.Toast;
 import com.pysiak.simple.clock.R;
+import com.pysiak.simple.clock.ui.alarm.AlarmPresenter;
+import com.pysiak.simple.clock.ui.alarm.AlarmView;
+import com.pysiak.simple.clock.ui.clock.ClockPresenter;
+import com.pysiak.simple.clock.ui.clock.ClockView;
+import com.pysiak.simple.clock.ui.stopwatch.StopWatchView;
+import com.pysiak.simple.clock.ui.timer.TimerView;
 
 import java.util.List;
 
 /**
- * Main class, launcher activity for application, implementation of {@link SimpleClockView}
+ * Main class, launcher activity for application, implementation of {@link ClockView}
  *
  * @author Yaroslav P.
  */
-public class SimpleClockActivity extends AppCompatActivity implements SimpleClockView {
+public class ClockActivity extends AppCompatActivity implements AlarmView, ClockView, StopWatchView, TimerView {
```

Comment:
only 1 interface -> SimpleClockView
All other need to be moved to appropriate packages - AlarmView to alarm, etc